workflow-templates/im-test-cypress.yml

```diff
@@ -59,7 +75,7 @@ jobs:
           dotnet-version: ${{ env.DOTNET_VERSION }}
 
       - name: dotnet build
-        working-directory: ./ # TODO: verify the root is the correct location for the solution file
+        working-directory: . # TODO: verify the root is the correct location of the solution file
```

Comment:
Will this plain dot get us mapped back to the git root?

Reply:
yeah and the auto-plan-and-cmment-on-pr workflow sets it that way.  I'm not sure why I had the ./ to begin with here.